lib/core/test/unit/Cardano/Wallet/Primitive/Slotting/Helpers.hs

```diff
@@ -0,0 +1,209 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- | Legacy slotting API functions.
+-- Used as a reference in tests.
+
+module Cardano.Wallet.Primitive.Slotting.Helpers
```

Comment:
Happy to see this moved to its own module, but is this not more `Legacy` or `Simple` than `Helpers`?